app/Base/configs/env.ts

```diff
@@ -7,3 +7,4 @@ export const isBeta = process.env.REACT_APP_ENVIRONMENT === 'beta';
 export const isAlpha = process.env.REACT_APP_ENVIRONMENT === 'alpha';
 export const isNightly = process.env.REACT_APP_ENVIRONMENT === 'nightly';
 export const isDev = !isBeta && !isAlpha && !isNightly;
+export const BACKEND_SERVER_URL = process.env.BACKEND_SERVER_URL as string;
```

Comment:
```suggestion
export const backendServerUrl = process.env.SERVER_URL as string;
```

We should also think about renaming this variable